rep-0003.rst

```diff
@@ -281,6 +281,68 @@ Build System Support:
 
 - Same as Indigo
 
+Melodic Morenia (May 2018 - May 2023)
+--------------------------------------
+Required Support for:
+
+- Ubuntu Artful (17.10)
+- Ubuntu Bionic (18.04)
+
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 28
+
+Architectures Supported:
+
+- amd64
+- arm32
+- arm64
+
+Targeted Languages:
+
+- C++11
+- Python 2.7
+
+  - Python >= 3.5 not required, but testing against it is recommended
```

Comment:
Since this is a soft testing requirement anyway, should we recommend `Python 3.6`? `Python 3.6` was released in December 2016, and will be receiving [security updates through December 2021.](https://www.python.org/dev/peps/pep-0494/#lifespan)

Reply:
Yes, this REP started with 3.6 but given that one of the targeted platforms (Debian Stretch) ships python 3.5 we changed it to 3.5 and above (https://github.com/ros-infrastructure/rep/pull/139#discussion_r146352231)

Reply:
Ah, fair enough. I hadn't gone through the collapsed comments :/ Thanks!